frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -32,9 +33,11 @@ export const getServiceBindingStatus = ({ FLAGS }: RootState): boolean =>
 export const getCheURL = (consoleLinks: K8sResourceKind[]) =>
   _.get(_.find(consoleLinks, ['metadata.name', 'che']), 'spec.href', '');
 
-export const getEditURL = (gitURL: string, gitBranch: string, cheURL: string) => {
+export const getEditURL = (vcsURI?: string, gitBranch?: string, cheURL?: string) => {
```

Comment:
I don't see a reason to make the `vicURI` parameter optional.
```suggestion
export const getEditURL = (vcsURI: string, gitBranch?: string, cheURL?: string) => {
```

Reply:
Updated.